web/src/alert_words.ts

```diff
@@ -33,24 +34,32 @@ const alert_regex_replacements = new Map([
     ["'", "(?:'|&#39;)"],
 ]);
 
-export function process_message(message) {
+export function process_message(message: Message): undefined {
```

Comment:
The return type seems to be more like `void` to me

Reply:
oops. revised.